contracts/ligo/src/english_auction/english_auction_tez_permit.mligo

```diff
@@ -37,10 +36,10 @@ let config_storage_with_permit (p, permit_storage : permit_config_param * permit
     | None -> 
       let auction_storage = configure_auction_storage(p.config, Tezos.sender, permit_storage.auction_storage) in
       {permit_storage with auction_storage = auction_storage; counter = permit_storage.counter + 1n}
-    | Some permit -> 
-        let unsigned : bytes = ([%Michelson ({| { SELF; ADDRESS; CHAIN_ID; PAIR; PAIR; PACK } |} : nat * bytes -> bytes)] (permit_storage.counter, permit.paramHash) : bytes) in
-        assert_msg (Crypto.check permit.signerKey permit.signature unsigned, "INVALID_PERMIT"); 
-        assert_msg (Crypto.blake2b (Bytes.pack p.config) = permit.paramHash, "Config params and param hash do not match");
+    | Some permit ->
+        let param_hash = Crypto.blake2b (Bytes.pack p.config) in
+        let unsigned : bytes = ([%Michelson ({| { SELF; ADDRESS; CHAIN_ID; PAIR; PAIR; PACK } |} : nat * bytes -> bytes)] (permit_storage.counter, param_hash) : bytes) in
```

Comment:
Consider using `(Tezos.self_address, Tezos.chain_id)` instead of raw Michelson

Reply:
Thanks, fixed in 39a95dd. `(Tezos.chain_id, Tezos.self_address)` is the correct ligo equivalent btw

Reply:
Full line being `let unsigned : bytes = Bytes.pack ((Tezos.chain_id, Tezos.self_address), (counter, param_hash)) in`